docs/content/patterns/review-and-submit/index.mdx

```diff
@@ -0,0 +1,305 @@
+
+In all situations ‘Review and submit’ tasks should enable the user to review all information they have provided before submitting a form.
+
+```jsx live
+<Stack gap={3}>
+	<DirectionLink direction="left" href="#">
+		Back
+	</DirectionLink>
+	<Stack gap={1.5}>
+		<H1>Title of multi-task process</H1>
+		<Text as="p" color="muted" fontSize="md">
+			The introductory paragraph provides context about the entire multi-task
+			form.
+		</Text>
+		<Text as="p" color="muted" fontSize="sm">
+			Takes around 15 minutes.
+		</Text>
+	</Stack>
+	<Details iconBefore label="More information about form">
+		<Text as="p">
+			This is a small paragraph of text that is supplementary to the main page
+			content.
+		</Text>
+	</Details>
+	<TaskList
+		items={[
+			{
+				href: '#',
+				label: 'Multi-page form 1',
+				message: 'Short description of the task',
+				status: 'todo',
+			},
+			{
+				href: '#',
+				label: 'Multi-page form 2',
+				message: 'Short description of the task',
+				status: 'todo',
+			},
+			{
+				href: '#',
+				label: 'Multi-page form 3',
+				message: 'Short description of the task',
+				status: 'blocked',
+			},
+		]}
+	/>
+</Stack>
+```
+
+## Editing from ‘Review and submit’
+
+To edit information on the ‘Review and Submit’ page, users can click the ‘Change’ link available under each summary list on the ‘Review and submit’ page.
+
+The ‘Edit’ page will appear as a sub-step of the ‘Review and submit’ Progress indicator item.
+
+Once the changes are saved, the user will be returned to the ‘Review and submit’ page, where a success message will be displayed above the updated information in the summary list.
+
+### Review and submit sub-step
+
+```jsx live
+<Columns gap={{ xs: 1, md: 3 }}>
+	<Column columnSpan={{ xs: 12, md: 4 }}>
+		<ProgressIndicator
+			activePath="#review/page-1"
+			items={[
+				{ label: 'Page 1', status: 'done', href: '#page-1' },
+				{ label: 'Page 2', status: 'done', href: '#page-2' },
+				{
+					items: [
+						{
+							label: 'Page 1 sub step',
+							href: '#review/page-1',
+						},
+					],
+					label: 'Review and submit',
+					status: 'started',
+					href: '#review',
+				},
+			]}
+		/>
+	</Column>
+	<Column columnSpan={{ xs: 12, md: 8 }}>
+		<Stack gap={3}>
+			<DirectionLink direction="left" href="#">
+				Back
+			</DirectionLink>
+			<Stack gap={1.5}>
+				<H1>
+					<Text
+						color="muted"
+						display="block"
+						fontSize="sm"
+						fontWeight="bold"
+						lineHeight="heading"
+					>
+						Title of multi page form
+					</Text>
+					Page 1 title
+				</H1>
+				<Text as="p" color="muted" fontSize="md">
+					Check your name and contact details.
+				</Text>
+			</Stack>
+			<FormStack>
+				<TextInput hideOptionalLabel label="Term" value="Description" />
+				<TextInput hideOptionalLabel label="Label text" value="text" />
```

Comment:
Why are these Label text? And shouldn't all the Terms have unique names (Term 1, Term 2 etc)?